joblib/numpy_pickle.py

```diff
@@ -89,7 +89,7 @@ def write_array(self, array, pickler):
                                                   'zerosize_ok'],
                                            buffersize=buffersize,
                                            order=self.order):
-                pickler.file_handle.write(chunk.tostring('C'))
+                pickler.file_handle.write(chunk.data)
```

Comment:
The related code in numpy is [here](https://github.com/numpy/numpy/blob/master/numpy/lib/format.py#L581) and [here](https://github.com/numpy/numpy/blob/master/numpy/lib/format.py#L589)